AUTHORS.txt

```diff
@@ -0,0 +1,6 @@
+The Spyder Project Contributors are composed of:
+
+* Yann Lanthony <https://github.com/yann-lty> (Original qtsass author).
```

Comment:
He should be listed separately above, just as you had it before

Reply:
woof

Reply:
Meow